src/styles/blocks/catalog.scss

```diff
@@ -16,10 +17,11 @@
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
-
-    &:not(:nth-child(4n)) {
-      margin-right: 48px;
+   
+    &:hover {
+      transition: $duraction;
```

Comment:
transition shoudn't be inside hover selector

set all properties which are being animated